model/pytorch/dffml_model_pytorch/utils/utils.py

```diff
@@ -34,6 +33,45 @@ def create_layer(layer_dict):
     return sequential_dict
 
 
+def create_network(network_dict):
+    sequential_network = nn.Sequential()
+    for name, layer in network_dict.items():
+        if isinstance(layer, dict):
+            parameters = {k: v for k, v in layer.items()}
+            layer_type = parameters.pop("layer_type")
```

Comment:
for each `layer_type` we are popping its 'label' and later assigning it to a valid `nn` model